### PR TITLE
Add relations

### PR DESCRIPTION
**Pull request by Ewoskyx**
### Class Relations

In this PR I did;

-Add Has many and belongs to relations between Classroom and Person
-Add Mant to many relations between Person and Book by Rental class
-Add additional play.rb file to represent project requirements

### CODE REVIEW

File: person.rb [hunk 1]
```diff
@@ -2,18 +2,19 @@
...
     @parent_permission = parent_permission
     @corrector = Corrector.new
+    @rentals = []
   end
 
-  def validate_name
-    @name = corrector.correct_name(@name)
+  def validate_name(name)
+    @name = @corrector.correct_name(name)
   end
```
Comment (er-contreras) [inline person.rb:19]: - [x] A method for adding rentals should be present.
- [x] The person itself should be sent as a parameter to create the new rental achieving the 'has-many' association
```
def add_rental(book, date)
    Rental.new(date, book, self)
end
```

**Review by er-contreras — CHANGES_REQUESTED** [review]
Hi @Ewoskyx, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

almost there

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: person.rb (hunk 1 above)
Comment (Ewoskyx) [reply person.rb:19]: Thank you, I will add it immediately.

**Review by FrederickMih — APPROVED** [review]
# APPROVED :rocket: 
Hi @Ewoskyx , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**